.github/workflows/test-acceptance.yaml

```diff
@@ -61,6 +61,10 @@ jobs:
         CYPRESS_DEFAULT_COMMAND_TIMEOUT: 40000
         CYPRESS_PAGE_LOAD_TIMEOUT: 120000
         CYPRESS_RETRIES: 3
+        CYPRESS_PASSWORD: password
```

Comment:
I can't see anywhere that `CYPRESS_PASSWORD` is used, is it necessary?

Reply:
It's not possible to access the PASSWORD environment variable in a cypress test however it is possible to access the CYPRESS_PASSWORD environment variable via `Cypress.env('PASSWORD')`.  It is therefore necessary.

Reply:
Ah, interesting. I think it might be worth adding something to `events/index.js` to read `process.env.PASSWORD` in to the Cypress env, similar to what we do for `KIT_TEST_DIR` already https://github.com/alphagov/govuk-prototype-kit/blob/main/cypress/events/index.js#L56. I think avoiding the duplication would be nicer and easier to use.

Reply:
I've implemented the above for password.